plugins/http_plugin/http_plugin.cpp

```diff
@@ -590,86 +590,91 @@ namespace eosio {
    void http_plugin::plugin_startup() {
 
       handle_sighup(); // setup logging
-
-      my->thread_pool.emplace( "http", my->thread_pool_size );
-
-      if(my->listen_endpoint) {
+      app().post(appbase::priority::high, [this] ()
+      {
+         my->thread_pool.emplace( "http", my->thread_pool_size );
```

Comment:
This should be moved into the try-catch.

Reply:
right, fixed

Reply:
fixed